### PR TITLE
completed example exercises and completed challenge moodly

### PR DESCRIPTION
**Pull request by goelvatsal**

### CODE REVIEW

File: section3/set10/challengeMoodly/main.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	args := os.Args[1:]
+
+	// if no user input received, then print usage msg
+	if len(os.Args) != 2 || args[0] == "" {
```
Comment (goel4607) [inline section3/set10/challengeMoodly/main.go:15]: The 2nd part of OR (||) can panic, as discussed correct the code such that the 2nd part of OR should be a safe check

File: section3/set10/challengeMoodly/main.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		fmt.Println("[your name]")
+		return
+	}
+
+	// error handling (ex: user types in int)
+	n, err := strconv.Atoi(args[0])
+	if err == nil {
+		fmt.Printf("%d is not a name.\n", n)
```
Comment (goel4607) [inline section3/set10/challengeMoodly/main.go:23]: I think it should return after this statement

File: section3/set10/challengeMoodly/main.go (hunk 1 above)
Comment (goelvatsal) [reply section3/set10/challengeMoodly/main.go:15]: there are no panics taking place when the code is run

File: section3/set10/challengeMoodly/main.go (hunk 2 above)
Comment (goelvatsal) [reply section3/set10/challengeMoodly/main.go:23]: added and fixed

**Pull request merged**